wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Follower Totals: update to new design

Fixes #11910 

This updates Insights _Follower Totals_ to use the two-column stat card.

To test:

- On a site with no Followers, go to Insights _Follower Totals_.
  - Verify `No data yet` is shown.
- On a site with Followers, go to Insights _Follower Totals_.
  - Verify the data is shown in two columns.

Update release notes:


---
<img width="400" alt="followers_no_data" src="https://user-images.githubusercontent.com/1816888/59631786-ebe49180-9105-11e9-9171-88a845857365.png">

---
<img width="400" alt="followers_total" src="https://user-images.githubusercontent.com/1816888/59631796-f3a43600-9105-11e9-9901-807ad8f45200.png">
